src/utility.js

```diff
@@ -410,7 +411,9 @@ function createItem(args, logger, notifier, requestKeys, lambdaContext) {
         break;
       case 'object':
       case 'array':
-        if (arg instanceof Error || (typeof DOMException !== 'undefined' && arg instanceof DOMException)) {
+        if (arg instanceof Error
+            || (typeof DOMException !== 'undefined' && arg instanceof DOMException)
+            || (typ === 'object' && arg.name && arg.message)) {
```

Comment:
This expression is long enough it probably merits being extracted into a function, especially since it appears twice.

Reply:
I noticed there was an `isError` function and I modified that instead.